site-src/guides/adapter-rollout.md

```diff
@@ -0,0 +1,126 @@
+# Adapter Rollout
+
+The goal of this guide is to demonstrate how to rollout a new adapter version.
+
+## **Requirements**
```

Comment:
I suggest changing this to Prerequisite.

Reply:
Updated.